src/js/components/PostTweet.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { postTweet } from '../services/timelineService';
+
+export default class PostTweet extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            message: '',
+            error: null,
+            isLoaded: false,
+            charCount: 0
+        };
+    }
+
+    handleInputChange(event){
```

Comment:
you could probably pass in the value directly based on the way you are referencing this function in your component

Reply:
agreed.  This is true given how you are handling the call to handleInputChange